crates/world/src/lib.rs

```diff
@@ -9,7 +9,22 @@ wasmtime::component::bindgen!({
     }
     "#,
     path: "../../wit",
-    async: true
+    async: true,
+    trappable_error_type: {
```

Comment:
It might be worth adding a comment above this indicating that this is a bit of a roundabout method of saying "the host implementations for these interfaces don't trap" but that's the intent of this configuration